x/oracle/keeper/prophecy.go

```diff
@@ -0,0 +1,135 @@
+package keeper
+
+import (
+	"errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Sifchain/sifnode/x/oracle/types"
+)
+
+// ProphecyLiftTime is used to clean outdated prophecy info from keeper
+const ProphecyLiftTime = 1000
```

Comment:
Per https://docs.google.com/document/d/1U420zg7tD7mAr1hlT2y6_BaqMepbnUM_cFfLz7zEfwo/edit#heading=h.k2oo0395vd1o

The ProphecyLiftTime should be 520,000